weave-js/src/components/Panel2/PanelPlot/PanelPlot2Inner.tsx

```diff
@@ -758,10 +786,10 @@ export const PanelPlot2Inner: React.FC<PanelPlotProps> = props => {
       concattedNonLineTable,
       concreteConfig.series,
       vegaReadyTables,
+      listOfTableNodes,
       vegaCols,
       flatResultNode,
-      weave.client.opStore,
-      listOfTableNodes,
+      weave,
```

Comment:
How about `const {opStore} = weave.client` at the top of the function and then just use `opStore` everywhere?

Reply:
done